docs/API.md

```diff
@@ -174,51 +200,52 @@ or simpler
 Sending and Receiving
 =====================
 
-The SRT API for sending and receiving is split into three categories: simple,
-rich, and for files only.
+The SRT API for sending and receiving is split into three categories: *simple*,
+*rich*, and *for files only*.
 
-The simple API includes: `srt_send` and `srt_recv` functions. They need only
+The **simple API** includes: `srt_send` and `srt_recv` functions. They need only
 the socket and the buffer to send from or receive to, just like system `read`
 and `write` functions.
 
-The rich API includes the `srt_sendmsg` and `srt_recvmsg` functions. Actually
+The **rich API** includes the `srt_sendmsg` and `srt_recvmsg` functions. Actually
 `srt_recvmsg` is provided for convenience and backward compatibility, as it is
 identical to `srt_recv`. The `srt_sendmsg` receives more parameters, specifically
-for messages. The `srt_sendmsg2` and `srt_recvmsg2` functions receive the socket, buffer, 
-and the `SRT_MSGCTRL` object, which is an input-output
-object specifying extra data for the operation.
+for messages. The `srt_sendmsg2` and `srt_recvmsg2` functions receive the socket, 
+buffer, and the `SRT_MSGCTRL` object, which is an input-output object specifying 
+extra data for the operation.
 
 Functions with the `msg2` suffix use the `SRT_MSGCTRL` object, and have the
-following interpretation (except `flags` and `boundary` that are reserved for
+following interpretation (except `flags` and `boundary` which are reserved for
 future use and should be 0):
 
 * `srt_sendmsg2`:
-    * msgttl: [IN] maximum time (in ms) to wait for successful delivery (-1: indefinitely)
-    * inorder: [IN] if false, the later sent message is allowed to be delivered earlier
-    * srctime: [IN] timestamp to be used for sending (0 if current time)
-    * pktseq: unused
-    * msgno: [OUT]: message number assigned to the currently sent message
+    * `msgttl`: [IN] maximum time (in ms) to wait for successful delivery (-1: indefinitely)
+    * `inorder`: [IN] if false, the later sent message is allowed to be delivered earlier
+    * `srctime`: [IN] timestamp to be used for sending (0 if current time)
+    * `pktseq`: unused
+    * `msgno`: [OUT] message number assigned to the currently sent message
 
 * `srt_recvmsg2`
-    * msgttl, inorder: unused
-    * srctime: [OUT] timestamp set for this dataset when sending
-    * pktseq: [OUT] packet sequence number (first packet from the message, if it spans multiple UDP packets)
-    * msgno: [OUT] message number assigned to the currently received message
+    * `msgttl`: inorder; unused
```

Comment:
Here were two fields mentioned in this entry. So, either leave as is, or split into two different enums.